test/oauth-server/service/getJWKs.go

```diff
@@ -13,5 +13,8 @@ func (requestHandler *RequestHandler) getJWKs(w http.ResponseWriter, r *http.Req
 			requestHandler.accessTokenJwkKey,
 		},
 	}
-	jsonResponseWriter(w, resp)
+	err := jsonResponseWriter(w, resp)
+	if err != nil {
+		writeError(w, err, http.StatusBadRequest)
```

Comment:
when w.Write fails you can only log error. it is not recommended to call twice w.WriteHeader